insitu/views/requirement_product_requirement.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core.urlresolvers import reverse
+from django.views.generic import CreateView, UpdateView, DeleteView
+
+from insitu import models
+from insitu import forms
+
+
+class RequirementProductRequirementAdd(CreateView):
```

Comment:
Actually, I would try to have only one add view and reuse the patterns in the urls. (requirement_product_requirement_patterns no longer will exist)